src/edu/gatech/cs2340/trydent/gui/TrydentDropdown.java

```diff
@@ -0,0 +1,67 @@
+package edu.gatech.cs2340.trydent.gui;
+
+import javafx.beans.value.ChangeListener;
+import javafx.collections.ObservableList;
+import javafx.scene.control.ChoiceBox;
+
+/**
+ * A basic dropdown box
+ * @author sparky
+ *
+ */
+@SuppressWarnings("rawtypes")
```

Comment:
Why not just parameterize ChoiceBox and ObservableList, rather than use raw types?